pytradfri/device.py

```diff
@@ -221,7 +220,8 @@ def set_predefined_color(self, colorname, *, index=0):
             color = COLORS[colorname.lower().replace(" ", "_")]
             return self.set_hex_color(color, index=index)
         except KeyError:
-            _LOGGER.debug('Could not match color name')
+            raise ColorError('Unable to match %s in COLOR_NAMES',
```

Comment:
Error message leaks implementation detail. I would go for "Invalid color specified: %s"